paramiko/client.py

```diff
@@ -279,6 +282,12 @@ def connect(
             The targets name in the kerberos database. default: hostname
         :param float banner_timeout: an optional timeout (in seconds) to wait
             for the SSH banner to be presented.
+        :param str pkcs11pin: If using PKCS11, this will be the pin of your
+            token or smartcard.
+        :param str pkcs11provider: If using PKCS11, this will be the provider
+            for the PKCS11 interface. Example: /usr/local/lib/opensc-pkcs11.so.
+        :param str pkcs11session: If using PKCS11 in a multithreaded
+            application you can share the session between threads.
```

Comment:
For most cases the pkcs11provider is all thats needed, very few users will need to use the pkcs11session argument.  pkcs11session is optional and only needed when used in a multithreaded application.